README.md

```diff
@@ -31,7 +35,7 @@ jobs:
       - name: Use HTTPS instead of SSH for Git cloning
         run: git config --global url.https://github.com/.insteadOf ssh://git@github.com/
       - name: Run GitHub Action for ORT
-        uses: oss-review-toolkit/ort-ci-github-action@v1
+        uses: oss-review-toolkit/ort-ci-github-action@main
```

Comment:
Same thing here, avoid @main, which is not seen very well. Consider always tag and pin down

Reply:
So keep with main in this case, but then for external actions, move to pinnning and then is ok